lib/kafka_ex/server_0_p_9_p_0.ex

```diff
@@ -55,6 +56,9 @@ defmodule KafkaEx.Server0P9P0 do
   defdelegate kafka_create_topics(requests, network_timeout, state),
     to: Server0P8P2
 
+  defdelegate kafka_delete_topics(requests, network_timeout, state),
```

Comment:
nit: delegating to the 0.8.2 server will cause a 0.9.0 client/broker to raise "DeleteTopic is not supported in 0.8.2 version of kafka", which might be a little confusing.

Reply:
I agree. I did as was done for the other defdelegates. I'm OK to change them all if you wish (I think it would be better to do so)